### ISSUE TITLE
Importing tensorflow fails.

### ISSUE THREAD

**Issue by acburigo** [tensorflow/tensorflow#9819]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10 Home
- **TensorFlow installed from (source or binary)**: Binary
- **TensorFlow version (use command below)**: I can't import, but the wheel file is "tensorflow_gpu-1.1.0-cp35-cp35m-win_amd64.whl (48.5MB)"
- **Bazel version (if compiling from source)**: -
- **CUDA/cuDNN version**: cuda_8.0.61_win10.exe / cudnn-8.0-windows10-x64-v5.1.zip
- **GPU model and memory**: NVIDIA GeForce GTX 1070
- **Exact command to reproduce**: import tensorflow

### Describe the problem
Importing tensorflow fails.

### Source code / logs
Installing tensorflow-gpu:
```
λ pip3 install tensorflow-gpu --upgrade
Collecting tensorflow-gpu
  Downloading tensorflow_gpu-1.1.0-cp35-cp35m-win_amd64.whl (48.5MB)
    100% |################################| 48.6MB 16.9MB/s
Requirement already up-to-date: wheel>=0.26 in c:\users\arthu\appdata\local\programs\python\python35\lib\site-packages (from tensorflow-gpu)
Requirement already up-to-date: protobuf>=3.2.0 in c:\users\arthu\appdata\local\programs\python\python35\lib\site-packages (from tensorflow-gpu)
Requirement already up-to-date: werkzeug>=0.11.10 in c:\users\arthu\appdata\local\programs\python\python35\lib\site-packages (from tensorflow-gpu)
Requirement already up-to-date: numpy>=1.11.0 in c:\users\arthu\appdata\local\programs\python\python35\lib\site-packages (from tensorflow-gpu)
Requirement already up-to-date: six>=1.10.0 in c:\users\arthu\appdata\local\programs\python\python35\lib\site-packages (from tensorflow-gpu)
Requirement already up-to-date: setuptools in c:\users\arthu\appdata\local\programs\python\python35\lib\site-packages (from protobuf>=3.2.0->tensorflow-gpu)
Requirement already up-to-date: packaging>=16.8 in c:\users\arthu\appdata\local\programs\python\python35\lib\site-packages (from setuptools->protobuf>=3.2.0->tensorflow-gpu)
Requirement already up-to-date: appdirs>=1.4.0 in c:\users\arthu\appdata\local\programs\python\python35\lib\site-packages (from setuptools->protobuf>=3.2.0->tensorflow-gpu)
Requirement already up-to-date: pyparsing in c:\users\arthu\appdata\local\programs\python\python35\lib\site-packages (from packaging>=16.8->setuptools->protobuf>=3.2.0->tensorflow-gpu)
Installing collected packages: tensorflow-gpu
Successfully installed tensorflow-gpu-1.1.0
```

Importing tensorflow:
```
λ python
Python 3.5.3 (v3.5.3:1880cb95a742, Jan 16 2017, 16:02:32) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow
Traceback (most recent call last):
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: The specified module could not be found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 41, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 21, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow_internal')
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow_internal'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import *
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 51, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 52, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: The specified module could not be found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 41, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 21, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow_internal')
  File "C:\Users\arthu\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow_internal'


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_problems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.
```

**Comment by drpngx** [comment]
Could you try the dependency walker on the tensorflow DLL to see what are the missing DLLs? I am suspecting CUDA.

**Comment by acburigo** [comment]
Where should this DLL be located in a default installation?

**Comment by SeanColombo** [comment]
I'm having the same problem as @acburigo 

> pip show tensorflow-gpu
...
Location:  C:\Users\[USER NAME]\appdata\local\conda\conda\envs\carnd-term1\lib\site-packages
...
There's a bunch of stuff there but I don't see a DLL. there's a "tensorflow" and "tensorflow_gpu-1.1.0.dist-info" directory in there.  "tensorflow" seems to be the location of the pywrap_tensorflow_internal.py that's generating the error.

@drpngx  do you know where there is supposed to be a tensorflow DLL?

**Comment by SeanColombo** [comment]
I was able to open the _python_tensorflow_internal.pyd with Dependency Walker and it complained about cudnn64_5.dll not being found.  I tried it on my command prompt and found that adding it to my path hadn't worked.

Fix for me was:
0. Type "cudnn64_5.dll" into command prompt and get "not recognized" to verify that DLL wasn't in my path.
1. Go back and add the location of the cudnn DLL to the path again (wherever you unzipped it to), making sure I clicked "OK" on the dialogs so that they close out correctly.
2. Close my open command prompts.
3. Open a new command prompt and type "cudnn64_5.dll", for me this opens NUnit since that is what opens DLLs on my setup.
4. Activate my anaconda environment & run again.
5. Bask in the fastness of GPU tensorflow!

Hope that helps the @acburigo
